components/builder-db/src/schema/jobs.rs

```diff
@@ -1,102 +0,0 @@
-table! {
```

Comment:
Similar to above, drop these db items now or wait for a later time and a different PR.